etc/config/pascal.amazon.properties

```diff
@@ -1,10 +1,9 @@
-compilers=&fpc
+compilers=&fpc:&delphi
 defaultCompiler=fpc302
 
 group.fpc.compilers=fpc260:fpc262:fpc264:fpc302
 group.fpc.options=@/opt/compiler-explorer/fpc/fpc.cfg -g -al
 group.fpc.demangler=/dev/null
-group.fpc.objdumper=/opt/compiler-explorer/gcc-7.2.0/bin/objdump
```

Comment:
No more objdump?

Reply:
ah, i think this was just for testing to make it run on my environment, will undo